internal/services/managedapplications/managed_application_resource_test.go

```diff
@@ -164,17 +164,17 @@ func TestAccManagedApplication_parameterValues(t *testing.T) {
 }
 
 func (ManagedApplicationResource) Exists(ctx context.Context, clients *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
-	id, err := parse.ApplicationID(state.ID)
+	id, err := applications.ParseApplicationID(state.ID)
 	if err != nil {
 		return nil, err
 	}
 
-	resp, err := clients.ManagedApplication.ApplicationClient.Get(ctx, id.ResourceGroup, id.Name)
+	resp, err := clients.ManagedApplication.ApplicationClient.Get(ctx, *id)
 	if err != nil {
-		return nil, fmt.Errorf("retrieving Managed Application %s (resource group: %s): %v", id.Name, id.ResourceGroup, err)
+		return nil, fmt.Errorf("retrieving Managed Application %s: %v", id.String(), err)
```

Comment:
_pedantic_ but for consistency:

```suggestion
		return nil, fmt.Errorf("retrieving %s: %v", *id, err)
```